src/obu.c

```diff
@@ -290,7 +290,7 @@ static avifBool parseSequenceHeaderColorConfig(avifBits * bits, avifSequenceHead
         header->av1C.chromaSubsamplingX = 1;
         header->av1C.chromaSubsamplingY = 1;
         header->yuvFormat = AVIF_PIXEL_FORMAT_YUV400;
-    } else if (header->colorPrimaries == AVIF_COLOR_PRIMARIES_SRGB && header->transferCharacteristics == AVIF_TRANSFER_CHARACTERISTICS_SRGB &&
+    } else if (header->colorPrimaries == AVIF_COLOR_PRIMARIES_BT709 && header->transferCharacteristics == AVIF_TRANSFER_CHARACTERISTICS_SRGB &&
```

Comment:
Yannis: This change is optional. Please let me know if you prefer to not change this.

Here we are checking for sRGB, so the use of `AVIF_COLOR_PRIMARIES_SRGB` is appropriate. But it may be clearer to use `AVIF_COLOR_PRIMARIES_BT709` here because one is likely to compare this code with the following in the AV1 spec:

```
    } else if ( color_primaries == CP_BT_709 &&
                transfer_characteristics == TC_SRGB &&
                matrix_coefficients == MC_IDENTITY ) {
        color_range = 1
        subsampling_x = 0
        subsampling_y = 0
    } else {
```

Reply:
I do not mind, in the WebP team we are more familiar with sRGB than BT709 but if libavif is closer to AV1 than to an image format in your opinion, that is fine with me.

Reply:
The purpose of this file (src/obu.c) is to parse the AV1 sequence heaser OBU. To check the correctness of the code in this file, one needs to compare it with the AV1 spec. That's why I think it would be clearer to match the AV1 spec in the constant's name.